classes/class-wc-connect-help-provider.php

```diff
@@ -249,7 +249,114 @@ protected function get_health_items() {
 		}
 
 		protected function get_services_items() {
-			return array();
+			$service_items = array();
+
+			$enabled_services = $this->service_settings_store->get_enabled_services();
+
+			if ( empty( $enabled_services ) ) {
+				$service_items[] =	(object) array(
+					'key' => 'wcc_services_empty',
+					'type' => 'text',
+					'class' => 'form_text_body_copy',
+					'value' => __( 'No services have been enabled', 'woocommerce' )
+				);
+
+				return $service_items;
+			}
+
+			foreach ( (array) $enabled_services as $enabled_service ) {
+				$indicator_key = "{$enabled_service->method_id}_{$enabled_service->instance_id}_1";
```

Comment:
Not needed. Removed in https://github.com/Automattic/woocommerce-connect-client/pull/398/commits/cb3acef1e09dbec4548df2b7836fd8815d9b23ce